packages/cli/assets/doc.md

```diff
@@ -39,6 +46,17 @@ To deploy this service, go to [this service page](https://marketplace.mesg.com/s
 
 ## Definitions
 
+{{#if configuration}}
+  {{#with configuration}}
+    {{#if env}}    
+### Environment Variables
+      {{#each env}}
+        - {{this}}
```

Comment:
The tab makes it look like a code block, because of that the `=` is replaced with `&#x3D;`

```suggestion
- {{this}}
```